nilearn/datasets/atlas.py

```diff
@@ -780,3 +780,84 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the Allen and MIALAB ICA atlas
+    (dated 2011).
+
+    The provided images are in MNI152 space.
+
+    Parameters
+    ----------
+    data_dir: str, optional
+        directory where data should be downloaded and unpacked.
+    url: str, optional
+        url of file to download.
+    resume: bool
+        whether to resumed download of a partly-downloaded file.
+    verbose: int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+        - "maps": T-maps of all 75 unthresholded components.
+        - "rsn28": T-maps of 28 RSNs included in E. Allen et al.
+        - "networks": string list containing the names for the 28 RSNs.
+        - "rsn_indices": dict[rsn_name] -> list of int, indices in the "maps"
+                         file of the 28 RSNs.
+        - "comps": The aggregate ICA Components.
+
+    References
+    ----------
+
+    E. Allen, et al, "A baseline for the multivariate comparison of resting
+    state networks," Frontiers in Systems Neuroscience, vol. 5, p. 12, 2011.
+
+    Notes
+    -----
+    Licence: unknown
+
+    See http://mialab.mrn.org/data/index.html for more information
+    on this dataset.
+    """
+    if url is None:
+        url = "http://mialab.mrn.org/data/hcp/"
+
+    dataset_name = "allen_rsn_2011"
+    keys = ("maps",
+            "rsn28",
+            "components")
```

Comment:
components -> comps or vice-versa. In Returns documentation you have written as comps.